.github/workflows/README.md

```diff
@@ -0,0 +1,246 @@
+# GitHub Actions Workflows
+
+This directory contains GitHub Actions workflows for the Antimetal System Agent project.
+
+## Workflows
+
+### 1. Linear Sync (`linear-sync.yml`)
+Synchronizes GitHub issues with Linear tasks.
+- **Triggers**: When issues are opened, closed, or reopened
+- **Actions**: 
+  - Creates Linear issues when GitHub issues are opened
+  - Updates Linear issue status when GitHub issues are closed/reopened
+
+### 2. Claude Code Review (`claude-code-review.yml`)
+Provides AI-powered code review using Claude on pull requests.
+- **Triggers**: Pull request events (opened, synchronize)
+- **Actions**: Reviews code changes and provides feedback
+
+### 3. Claude Code (`claude.yml`)
+Responds to mentions of Claude in issue comments.
+- **Triggers**: Issue comments containing "@claude"
+- **Actions**: Processes requests and provides assistance
```

Comment:
Github Actions has a description field https://docs.github.com/en/actions/reference/metadata-syntax-reference#description. Think it would be better to put it there because it will be more likely to be kept up to date